### PR TITLE
Build.IsLocked tests for active docker containers named build-B*. Bui…

### PR DESCRIPTION
**Pull request by nzoschke**
…ldCreate controller test for lock and returns 403

### CODE REVIEW

**Comment by nzoschke** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;pLX29Kdp&#x2F;234-improve-locking-around-builds-and-agent-image-cleaning"> Improve locking around builds and agent image cleaning</a>

**Comment by csquared** [comment]
LGTM

File: api/controllers/builds.go [hunk 1]
```diff
@@ -66,6 +66,10 @@ func BuildGet(rw http.ResponseWriter, r *http.Request) *httperr.Error {
 func BuildCreate(rw http.ResponseWriter, r *http.Request) *httperr.Error {
 	build := models.NewBuild(mux.Vars(r)["app"])
 
+	if build.IsLocked() {
```
Comment (csquared) [inline api/controllers/builds.go]: for me, a `lock` would imply actually setting and unsetting a lock. this check seems here to be more around if it is `busy`, `running`, etc. so I think there's a handful of better names than `IsLocked` for this one.  More to consider than a must change.

Comment (nzoschke) [inline api/controllers/builds.go]: renamed to IsRunning. The comments were already in that vein

**Pull request merged**